examples/resources/awscc_cassandra_table/table.tf

```
@@ -0,0 +1,12 @@
+resource "awscc_cassandra_keyspace" "awscc_cassandra_keyspace_example" {
+  keyspace_name = "awscc_cassandra_keyspace_example"
+}
+
+resource "awscc_cassandra_table" "awscc_cassandra_table_example" {
+  keyspace_name = "awscc_cassandra_keyspace_example"
+  partition_key_columns = [{
+    column_name = "Message"
+    column_type = "ASCII"
+  }]
+  depends_on = [awscc_cassandra_keyspace.awscc_cassandra_keyspace_example]
+}
```

```suggestion
resource "awscc_cassandra_table" "awscc_cassandra_table_example" {
  keyspace_name = awscc_cassandra_keyspace.awscc_cassandra_example.id
  partition_key_columns = [{
    column_name = "Message"
    column_type = "ASCII"
  }]
}
```

Instead of explicit `depends_on` dependencies, let Terraform resolve it automatically by referring to the keyspace id